index.android.js

```diff
@@ -5,8 +5,12 @@ import {
 } from 'react-native';
 import EStyleSheet from 'react-native-extended-stylesheet'
 import ProjectList from './ProjectList'
+import GoogleAnalytics from 'react-native-google-analytics-bridge'
 import theme from './theme'
 
+GoogleAnalytics.setTrackerId('UA-1224199-62')
```

Comment:
Normally I think this would preferably use an environment variable, but unfortunately that's not available in a mobile context.  What if I just pulled it out to define in a separate file so it's at least abstracted out?